--- hello-world.rs/build.rs
+++ hello-world.rs/build.rs
@@ -5,7 +5,7 @@
 fn main() {
     println!("cargo:rerun-if-changed=hello-world.rs.rc");
     println!("cargo:rerun-if-changed=hello-world.rs.exe.manifest");
-    if env::var("CARGO_CFG_TARGET_FAMILY").unwrap() == "windows" {
+    if false && env::var("CARGO_CFG_TARGET_FAMILY").unwrap() == "windows" {
         let manifest_template = fs::read_to_string("hello-world.rs.exe.manifest").unwrap();
         let arch = match &*env::var("CARGO_CFG_TARGET_ARCH").unwrap() {
             "x86" => "x86",
